src/super_gradients/training/utils/detection_utils.py

```diff
@@ -385,20 +386,12 @@ def _draw_box_title(
         color = color_mapping[class_id]
         class_name = class_names[class_id]
 
-        # Draw the box
-        image_np = cv2.rectangle(image_np, (x1, y1), (x2, y2), color, box_thickness)
-
-        # Caption with class name and confidence if given
-        text_color = (255, 255, 255)  # white
-
         if is_target:
             title = f"[GT] {class_name}"
-        if not is_target:
+        else:
```

Comment:
looks like it will still print "[Pred]"

Reply:
When using `_draw_box_title` yes because we do use in in visualization that plots both predictions and ground truth.
In the case of predict, we don't call this `_draw_box_title` method anymore, and instead directly call `draw_bbox`